src/core/ComboBox/ComboBox.tsx

```diff
@@ -36,10 +36,6 @@ export type ComboBoxProps<T> = {
    * Message shown below the combobox.
```

Comment:
Maybe mention that `Message` component is recommended to be used for this prop

Reply:
mentioned